core/object/object.cpp

```diff
@@ -911,30 +911,34 @@ Variant Object::call_const(const StringName &p_method, const Variant **p_args, i
 	return ret;
 }
 
-void Object::notification(int p_notification, bool p_reversed) {
-	if (p_reversed) {
-		if (script_instance) {
-			script_instance->notification(p_notification, p_reversed);
-		}
-	} else {
-		_notificationv(p_notification, p_reversed);
```

Comment:
This appears to be lost in the new version. Shouldn't it appear at the top of  `_notification_forward` and the end of `_notification_backward`?

Reply:
`_notificationv` was removed because it's replaced by `_notification_forward` and `_notification_backward` (depending on the boolean argument).